src/common/impl_registration.hpp

```diff
@@ -203,13 +203,13 @@
 
 // Primitive CPU ISA section is in src/cpu/platform.hpp
 
-#if BUILD_PRIMITIVE_GPU_ISA_ALL || BUILD_GEN9
+#if BUILD_PRIMITIVE_GPU_ISA_ALL
 #define REG_GEN9_ISA(...) __VA_ARGS__
```

Comment:
I'd double-check if we want to keep the functionality guarded by these macros. We might save some library spaces by removing all related functionality altogether.